pom.xml

```diff
@@ -254,7 +259,7 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>${argLine} -Djava.awt.headless=true -Dfile.encoding=UTF-8</argLine>
+          <argLine>${argLine} -Djava.awt.headless=true -Dfile.encoding=UTF-8 -Duser.language=en -Duser.country=US</argLine>
```

Comment:
@paulodamaso does this line required for origin task? If yes, it would be more clear to use `<systemPropertyVariables>` configuration instead of `argLine`, what do you think? https://maven.apache.org/plugins-archives/maven-surefire-plugin-2.12.4/examples/system-properties.html
```xml
<configuration>
  <systemPropertyVariables>
    <file.encoding>UTF-8</file.encoding>
    <user.language>en</user.language>
    <user.country>US</user.country>
  <systemPropertyVariables>
</configuration>```

Reply:
@g4s8 Not really... This line is required for `Takes` running on windows build in my locale (pt-BR) and it may cause problems in other locales too (just like https://github.com/zerocracy/farm/issues/1041). I agree with you that it is clearer to use `<systemPropertyVariables>` so I'll change it.